learners/setup.md

```diff
@@ -142,6 +142,22 @@ These installation steps could ask you `? Do you want to continue (Y/n)` write `
 
 ::::::::::::::::::::::::::::: spoiler
 
+### do you get an error with EpiNow2?
+
+Windows users will need a working installation of `Rtools` in order to build the package from source. `Rtools` is not an R package, but a software you need to download and install. We suggest you to follow [these steps](http://jtleek.com/modules/01_DataScientistToolbox/02_10_rtools/#1):
+
+1. Download the `Rtools` installer from <https://cran.r-project.org/bin/windows/Rtools/>
+2. Install `Rtools` with default selections
+
+To verify your installation follow these two steps:
+
+3. Install `{devtools}` -->  `install.packages("devtools")`
+4. Verify Rtools installation with --> `devtools::find_rtools()`
```

Comment:
I would ask them to first check if they have `rtools` installed. They can do so by simply using Windows search across their system or use devtools (optional). For the latter, they need to have devtools installed. 

So, ideally, they should try step 4 first. If it says "Error in loadNamespace(x) : there is no package called ‘devtools’"), they should install devtools and try again. If the result is `FALSE`, then they should do (1). So, essentially, the order here is reversed. It should be (4) -> (3) -> (2) -> (1) and maybe (4) again.

Reply:
Thanks! I rarranged the steps in https://github.com/epiverse-trace/tutorials-middle/pull/61/commits/6721a9aa4fd12d03ae6204b18ddc9d940972976c